dirsrvtests/tests/suites/retrocl/basic_test.py

```diff
@@ -0,0 +1,120 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+
+import logging
+import ldap
+import time
+import pytest
+from lib389.topologies import topology_st as topology
+from lib389.plugins import RetroChangelogPlugin
+from lib389._constants import *
+from lib389.utils import *
+from lib389.tasks import *
+
+USER1_DN = 'uid=user1,'+ DEFAULT_SUFFIX
+USER2_DN = 'uid=user2,'+ DEFAULT_SUFFIX
+USER_PW = 'password'
+ATTR_HOMEPHONE = 'homePhone'
+ATTR_CARLICENSE = 'carLicense'
+
+log = logging.getLogger(__name__)
+
+def test_retrocl_exclude_attr_add(topology):
+    """ Test exclude attribute feature of the retrocl plugin for add operation
+
+    :id: todo
+
+    :setup: Standalone instance
+
+    :steps:
+        1. Enable Retro Changelog
+        2. Enable dynamic plugins
+        3. Add an entry
+        4. Ensure entry attrs are in the changelog
+        5. Exclude an attr
+        6. Add another entry
+        7. Ensure entry attrs are not in the changelog
+
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+        7. Success
+    """
+    st = topology.standalone
+
+    # Enable RetroChangelog plugin.
+    rcl = RetroChangelogPlugin(st)
+    rcl.enable()
+
+    # Set the default targetid.
+    rcl.replace('nsslapd-attribute', 'nsuniqueid:targetUniqueId')
+    # Enable dynamic plugins.
+    rcl.replace('nsslapd-dynamic-plugins', 'on')
+
+    st.restart()
+
+    log.info('Add entry ' + USER1_DN)
+    try:
+        st.add_s(Entry((USER1_DN, {'objectclass': "top person organizationalPerson inetOrgPerson shadowAccount".split(),
```

Comment:
Please, avoid using 'Entry' object as it'll be deprecated at some point.

We should stick to DSLdapObject approach in most cases.

If you need a user, you can use UserAccounts.create()